### PR TITLE
LPS-93158 Add integration tests

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by jeyvison** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v1_0/test/DataRecordCollectionResourceTest.java [hunk 1]
```diff
@@ -147,6 +200,87 @@ protected DataRecordCollection randomIrrelevantDataRecordCollection()
...
+				siteId, keywords, Pagination.of(1, 2));
+
+		Assert.assertEquals(1, page.getTotalCount());
+
+		assertEqualsIgnoringOrder(
+			Arrays.asList(dataRecordCollection),
+			(List<DataRecordCollection>)page.getItems());
+		assertValid(page);
+
+		dataRecordCollectionResource.deleteDataRecordCollection(
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v1_0/test/DataRecordCollectionResourceTest.java]: Whye are we deleting the data record collection at the end of the test?

**Pull request closed without merging**